### ISSUE TITLE
Form > Redirect : Add native support to "Open in New Tab" and apply NoFollow

### ISSUE THREAD

**Issue by Iconised** [elementor/elementor#14181]
## Prerequisites
- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
Whilst using the Redirect function on the Form widget, there is no sensible solution to opening a redirected URL in a new tab. Past suggestions have been to add jQuery to interact with the widget, but this needs to be added as native functionality.

**Describe the solution you'd like**
The Redirect controls need to be natively updated to use the URL control set and include new tab and no follow selections.

**Describe alternatives you've considered**
This can be configured in the majority of other builders without requiring additional external code.

**Additional context**
Previously raised here without solution - https://github.com/elementor/elementor/issues/6974. In order to make this function competitive, this should be included natively.

**Comment by fbcudaipur** [comment]
I agree. This has to be a native feature as almost all other builders have this. Sad that this wasn't implemented with the latest Elementor or Elementor Pro updates.

**Comment by neylwalecki** [comment]
+1

**Comment by pedraohenrique** [comment]
+1

**Comment by neylwalecki** [comment]
+1 (2023 edition)

**Comment by lapo18** [comment]
+1

**Comment by venkateshwarans** [comment]
+1

**Comment by arorabhishek** [comment]
And years later with so many updates this is still not implemented. Implementing a Jquery is the only solution yet, with forms now being a default feature this more than ever an important update. Not sure what's stopping them!

**Comment by neylwalecki** [comment]
#forever-wishing-and-waiting-this
